miio/tests/test_chuangmi_ir.py

```diff
@@ -0,0 +1,62 @@
+from unittest import TestCase
+from miio import ChuangmiIr
+from miio.chuangmi_ir import ChuangmiIrException
+from .dummies import DummyDevice
+import pytest
+import base64
+
+PROSONIC_POWER_ON = 'Z6VPAAUCAABgAgAAxQYAAOUIAACUEQAAqyIAADSeAABwdQEAAAAAAAA' \
+                    'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABFEBAQEBAQEBAgICAgIC' \
+                    'AgEBAgECAQEBAQIBAgECAgICBgNXA1cDUA'
+
+
+class DummyChuangmiIr(DummyDevice, ChuangmiIr):
+    def __init__(self, *args, **kwargs):
+        self.state = {}
+        self.return_values = {
+            'miIO.ir_learn': lambda x: True,
+            'miIO.ir_read': lambda x: True,
+            'miIO.ir_play': lambda x: self._ir_play_input_validation(x),
+        }
+        super().__init__(args, kwargs)
+
+    @staticmethod
+    def _ir_play_input_validation(self, payload):
```

Comment:
I think this definition is wrong for staticmethod, if you remove `self` it should start working :-)